flight_phase_files/HALO_RF12_20200211_info.yaml

```diff
@@ -27,63 +32,301 @@ remarks:
+    - HALO-0211_s56
+    - HALO-0211_s57
+    - HALO-0211_s58
+    - HALO-0211_s59
+    - HALO-0211_s60
+    BAD:
+    - HALO-0211_s45
+    UGLY:
+    - HALO-0211_s07
+    - HALO-0211_s08
+    - HALO-0211_s36
+- kinds:
+  - circle
+  name: circle 1
+  irregularities: []
+  segment_id: HALO-0211_c1
+  start: 2020-02-11 12:57:31
+  end: 2020-02-11 13:54:16
+  dropsondes:
+    GOOD:
+    - HALO-0211_s01
+    - HALO-0211_s02
+    - HALO-0211_s03
+    - HALO-0211_s04
+    - HALO-0211_s05
+    - HALO-0211_s06
+    - HALO-0211_s09
+    - HALO-0211_s10
+    - HALO-0211_s11
+    - HALO-0211_s12
+    BAD: []
+    UGLY:
+    - HALO-0211_s07
+    - HALO-0211_s08
+- kinds:
+  - circle_break
+  name: circle break 1
+  irregularities: []
+  segment_id: HALO-0211_cb1
+  start: 2020-02-11 13:54:16
+  end: 2020-02-11 14:11:02
+  dropsondes:
+    GOOD: []
+    BAD: []
+    UGLY: []
+- kinds:
+  - circle
+  name: circle 2
+  irregularities: []
+  segment_id: HALO-0211_c2
+  start: 2020-02-11 14:11:02
+  end: 2020-02-11 15:09:18
+  dropsondes:
+    GOOD:
+    - HALO-0211_s13
+    - HALO-0211_s14
+    - HALO-0211_s15
+    - HALO-0211_s16
+    - HALO-0211_s17
+    - HALO-0211_s18
+    - HALO-0211_s19
+    - HALO-0211_s20
+    - HALO-0211_s21
+    - HALO-0211_s22
+    - HALO-0211_s23
+    - HALO-0211_s24
+    BAD: []
+    UGLY: []
+- kinds:
+  - circle_break
+  name: circle break 2
+  irregularities: []
+  segment_id: HALO-0211_cb2
+  start: 2020-02-11 15:09:18
+  end: 2020-02-11 15:26:51
+  dropsondes:
+    GOOD: []
+    BAD: []
+    UGLY: []
+- kinds:
+  - circle
+  name: circle 3
+  irregularities: []
+  segment_id: HALO-0211_c3
+  start: 2020-02-11 15:26:51
+  end: 2020-02-11 16:26:45
+  dropsondes:
+    GOOD:
+    - HALO-0211_s25
+    - HALO-0211_s26
+    - HALO-0211_s27
+    - HALO-0211_s28
+    - HALO-0211_s29
+    - HALO-0211_s30
+    - HALO-0211_s31
+    - HALO-0211_s32
+    - HALO-0211_s33
+    - HALO-0211_s34
+    - HALO-0211_s35
+    BAD: []
+    UGLY:
+    - HALO-0211_s36
+- kinds:
+  - circle_break
+  name: circle break 3
+  irregularities: []
+  segment_id: HALO-0211_cb3
+  start: 2020-02-11 16:26:45
+  end: 2020-02-11 16:38:51
+  dropsondes:
+    GOOD: []
+    BAD: []
+    UGLY: []
+- kinds:
+  - circle
+  name: circle 4
+  irregularities: []
+  segment_id: HALO-0211_c4
+  start: 2020-02-11 16:38:51
+  end: 2020-02-11 17:40:09
+  dropsondes:
+    GOOD:
+    - HALO-0211_s37
+    - HALO-0211_s38
+    - HALO-0211_s39
+    - HALO-0211_s40
+    - HALO-0211_s41
+    - HALO-0211_s42
+    - HALO-0211_s43
+    - HALO-0211_s44
+    - HALO-0211_s46
+    - HALO-0211_s47
+    - HALO-0211_s48
+    BAD:
+    - HALO-0211_s45
+    UGLY: []
+- kinds:
+  - circle_break
+  name: circle break 4
+  irregularities: []
+  segment_id: HALO-0211_cb4
+  start: 2020-02-11 17:40:09
+  end: 2020-02-11 17:53:00
+  dropsondes:
+    GOOD: []
+    BAD: []
+    UGLY: []
+- kinds:
+  - circle
+  name: circle 5
+  irregularities:
+  - First sonde 63 sec, to adjust for early exit
```

Comment:
53 seconds according to report

Reply:
I shifted the circle to start the normal 60s before the first sonde and with the new dropsonde definition this is actually possible. Therefore, the `irregularities` statement is unnecessary and I deleted it.